### ISSUE TITLE
Reintegrate WearOS codebase into main branch of LAMP-core-android

### ISSUE THREAD

**Issue by avaidyam** [BIDMCDigitalPsychiatry/LAMP-platform#532]
_(See discussion from issue #476.)_

> @saranya-sajeev: Wear os app and phone app are separate code bases. So we have to create two folders under Lamp- core repository, one for WearOs app and other for Phone app. Shall we proceed with this?
>
> @avaidyam: The WearOS codebase should NOT be separate. It is a module within the same Gradle project as the mobile app. It will have a separate folder under the `LAMP-core-android` repository, yes, just like how the Apple Watch app has its own folder in the `LAMP-core-ios` repository.

**Comment by saranya-sajeev** [comment]
@avaidyam ok.

**Comment by michaelmenon** [comment]
Saranya is going to working on this from tomorrow after the Telephony updates are done.

**Comment by saranya-sajeev** [comment]
@avaidyam do you need to keep wear os in a separate folder in the master branch or do you want it within the mindLamp source?

**Comment by avaidyam** [comment]
@saranya-sajeev It should a separate module (i.e. folder, with its own `build.gradle`) but in the `master` branch. [Please follow the instructions provided by Google here if you're still unsure.](https://developer.android.com/training/wearables/get-started/creating)

**Comment by ZCOEngineer** [comment]
Ok Aditya, @saranya-sajeev  will comment on this one the workflow is corrected.

**Comment by saranya-sajeev** [comment]
@avaidyam I will integrate wear os code in master right after you create the release. Otherwise, there may be issues with code references.

**Comment by saranya-sajeev** [comment]
@avaidyam WearOs code integrated to master branch.

**Comment by avaidyam** [comment]
So if I make a new release for Android, it will also appear available for WearOS?

**Comment by saranya-sajeev** [comment]
@avaidyam yes it should work, but yesterday it is not worked. May be any issues with wear os workflow. We will check and update you.

**Comment by saranya-sajeev** [comment]
@aditya wear os workflow is not shown under actions tab? Do you have any suggestions?

**Comment by avaidyam** [comment]
@saranya-sajeev Unfortunately I don't. It should be included in the Gradle build process if you've added it as a submodule correctly. Please consult the Android/wearOS documentation.

**Comment by saranya-sajeev** [comment]
@avaidyam We have created a separate workflow file for WearOs.  We will merge it to the main workflow once it is completed. Now it is showing following error  and we are working on it. We have checked the syntax on [yamllint](http://www.yamllint.com/) and it says it is valid.

Error: .github#L1
every step must define a `uses` or `run` key

**Comment by saranya-sajeev** [comment]
@avaidyam We have fixed the jar signer issue, it was due to the signing configuration already defined in the build.gradle file.

**Comment by saranya-sajeev** [comment]
@avaidyam Now we got below issue. I thnik it is due to same package name used for both mindLamp and wearos. We wil check on it. 
Error: Error: APK specifies a version code that has already been used.

**Comment by saranya-sajeev** [comment]
@avaidyam wearosa pp and minLamp should have same package name and so we have added an empty commit in wear os workflow to make the version code different. Currently we are getting below error and we are working on it.

You cannot rollout this release because it does not allow any existing users to upgrade to the newly added APKs.
